source/features/hide-in-case-you-missed-notifications.js

```diff
@@ -0,0 +1,7 @@
+export default function () {
+	$('.with-contextTweet')
+		.filter(function () {
+			return this.textContent.toLowerCase().indexOf('in case you missed') > -1;
```

Comment:
You can use `.includes()` instead of `.indexOf()`.

Reply:
Fixed!